src/components/Tabs/__snapshots__/Tabs.test.jsx.snap

```diff
@@ -0,0 +1,144 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders without crashing 1`] = `
+.glamor-8,
+[data-glamor-8] {
+  border-bottom: 1px solid rgba(0, 3, 6, 0.12);
+}
+
+.glamor-7,
+[data-glamor-7] {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  align-items: center;
+  overflow-x: auto;
+  overflow-y: hidden;
+  -webkit-overflow-scrolling: touch;
+  mask-image: -webkit-linear-gradient(to left, transparent, #000000 0.75rem);
+  mask-image: -moz-linear-gradient(to left, transparent, #000000 0.75rem);
+  mask-image: linear-gradient(to left, transparent, #000000 0.75rem);
+  -webkit-box-align: center;
+  -webkit-align-items: center;
+  -webkit-mask-image: -webkit-linear-gradient(to left, transparent, #000000 0.75rem);
+  -webkit-mask-image: -moz-linear-gradient(to left, transparent, #000000 0.75rem);
```

Comment:
Heh... funny the way this ended up (mixing the -moz and -webkit prefixes). Effectively a bug in ... glamorous I guess? Not that big of a deal cause they will probably just be ignored.